### ISSUE TITLE
Doc - missing proxyName & proxyPort & scheme parameters

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#3179]
IIRC these were mandatory when installing with tomcat.
Seems they are missing in https://github.com/georchestra/georchestra/blob/master/docs/setup/tomcat.md#configure-connector

**Comment by fvanderbiest** [comment]
See also 2040#issuecomment-389167070

**Comment by landryb** [comment]
they're not mandatory for all tomcats, but i know i needed it for georchestra & geoserver tomcats... and that also lead to issues for mapfishprint for mapstore (cf georchestra/mapstore2-georchestra#243)

**Comment by fvanderbiest** [comment]
Witnessed today, missing  
```
scheme="https"
proxyName="www.mydomain.org"
proxyPort="443"
```
on tomcat-geoserver after an ansible install with tomcat 9

**Comment by fvanderbiest** [comment]
Fixed from 20.0 on with e0b6f71b44cff612ac9fb6dcee47471f05a4bb2e

**Comment by fvanderbiest** [comment]
cc @landryb @pierrejego the ansible recipe will have to take this into account too
